src/main.cpp

```diff
@@ -1,227 +1,49 @@
-//by Alexander Prosser
-
+// snake
+// TODO move from ncurses to low-level system libraries (windows.h and unistd.h)
+#include <ncurses/ncurses.h>
 #include <iostream>
-#include <conio.h>
-#include <windows.h>
-using namespace std;
-
-bool gameOver;
-
-const int width = 50;
-const int height = 20;
-
-int x, y, fruitX, fruitY, score;
 
-int tailX[100], tailY[100];
-int nTail;
-
-enum eDirecrion {STOP = 0, LEFT, RIGHT, UP, DOWN};
-eDirecrion dir;
-
-int colorBarrier = 8;
-int colorSnake = 10;
-int colorFruit = 4;
-
-void setup()
-{
-    gameOver = false;
-    dir = STOP;
+#include "snake.h"
 
-    x = width / 2;
-    y = height / 2;
+/*
+ * // TODO rework field like this
+ * field:
+ *
+ * ############# SNEK
+ * #           # by MCWertGaming
```

Comment:
We have to talk about that one... xD

Reply:
I don't understand, if you mean, that only my name is listed there (like in the license btw) create a feature and add something and you will be listed there as well xD